heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of the Alien 1.2.2 game port for BeOS"
+HOMEPAGE="http://hota.sourceforge.net"
+COPYRIGHT="2004 Gil Megidish"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="git://github.com/ytmytm/beos-heartofthealien"
+SOURCE_DIR="heartofthealien"
+PATCHES="heartofthealien-1.2.2.patch"
+
+ARCHITECTURES="x86_gcc2 ?x86 ?x86_64"
+
+PROVIDES="
+	heartofthealien = $portVersion
+	cmd:heartofthealien = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libsdl
+	lib:libsdl_mixer
+	lib:libsmpeg
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libsdl
+	devel:libsdl_mixer
+	devel:libsmpeg
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
+	cmd:touch
+	"
+
+BUILD()
+{
+	cd src
+	make 
+}
+
+INSTALL()
+{
+	mkdir -p $appsDir
+	cp -af /sources/heartofthealien/src* $appsDir/heartofthealien
```

Comment:
Do we need to copy all the source files too? Because you do that here afaik.

Reply:
Sure. Because all file in src folder need for make alien binary file.